sdk/playwrighttesting/microsoft-playwright-testing/src/common/entraIdAccessToken.ts

```diff
@@ -21,9 +21,13 @@ class EntraIdAccessToken {
     try {
       coreLogger.info("Fetching entra id access token");
       const accessToken = await this._credential!.getToken(EntraIdAccessTokenConstants.SCOPE);
-      if (!accessToken) throw new Error("Entra id access token is null");
+      if (!accessToken){
+        process.stdout.write("\nEntra id access token is null.");
+        throw new Error("Entra id access token is null");
+      } 
       if (accessToken.token === this.token) {
         // azure identity library can fetch the same token again from cache. 10 mins before expiry, it allows token refresh
+        process.stdout.write("\nCached access token is returned, will be retried again.");
```

Comment:
was this requirement to stdout ? check with @Sid200026

Reply:
@pratiksharma23